app/src/features/surveys/list/SurveysListPage.tsx

```diff
@@ -60,14 +52,8 @@ const SurveysListPage = () => {
         )}
       />
       <Divider></Divider>
-      <Box p={3}>
-        <Paper variant="outlined">
-          <SurveysList
-            projectId={projectContext.projectId}
-            surveysList={surveysListDataLoader.data || []}
-            codes={codes}
-          />
-        </Paper>
+      <Box px={1}>
+        <SurveysList projectId={projectContext.projectId} surveysList={surveys || []} codes={codes} />
```

Comment:
We should refactor this component so that `codes` is no longer passed as a prop, since we now consume it as a context instead

Reply:
done. Would this make sense to do with the survey list items? 

With the new context pattern should we avoid passing data in via props unless we've got a reason to? @NickPhura @curtisupshall

Reply:
Ideally, ya! But for now it's definitely not urgent :)